packages/vm/tests/BlockchainTestsRunner.ts

```diff
@@ -50,7 +42,7 @@ export default async function runBlockchainTest(options: any, testData: any, t:
   })
 
   if (validatePow) {
-    ;(blockchain.ethash as any).cacheDB = cacheDB
+    blockchain.ethash!.cacheDB = cacheDB
```

Comment:
Ah that `!` is so cool...gotta start using that.

Reply:
Oh yeah, had to look this up [here](https://stackoverflow.com/questions/42273853/in-typescript-what-is-the-exclamation-mark-bang-operator-when-dereferenci) again, this would have been super useful along the `TypeScript` transition of the client, remember that I stumbled upon this several times (can't find the references though any more).

Anyhow.

Can still be improved.